.gitignore

```diff
@@ -0,0 +1,6 @@
+# .gitignore
+node_modules/
+.hintrc
+package-lock.json
+package.json
+.stylelintrc
```

Comment:
- [ ] Please remove these files from .gitignore, they are not meant to be ignored. Only node_modules should be ignored.